WaveTournament/index.js

```diff
@@ -0,0 +1,162 @@
+let socket = new ReconnectingWebSocket("ws://127.0.0.1:24050/ws");
+let mapid = document.getElementById('mapid');
+
+// NOW PLAYING
+let mapContainer = document.getElementById("mapContainer");
+let mapTitle = document.getElementById("mapTitle");
+let mapDifficulty = document.getElementById("mapDifficulty");
+
+// TEAM OVERALL SCORE
+let teamBlueName = document.getElementById("teamBlueName");
+let teamRedName = document.getElementById("teamRedName");
+let scoreNowBlue = document.getElementById("scoreNowBlue");
+let scoreNowRed = document.getElementById("scoreNowRed");
+let scoreMaxBlue = document.getElementById("scoreMaxBlue");
+let scoreMaxRed = document.getElementById("scoreMaxRed");
+
+// For Star Visibility
+let scoreBlue = document.getElementById("scoreBlue");
+let scoreRed = document.getElementById("scoreRed");
+let teamBlue = document.getElementById("teamBlue");
+let teamRed = document.getElementById("teamRed");
+
+// TEAM PLAYING SCORE
+let playScoreBlue = document.getElementById("playScoreBlue");
+let playScoreRed = document.getElementById("playScoreRed");
+
+// Graphic components
+let bottom = document.getElementById("bottom");
+
+socket.onopen = () => {
+    console.log("Successfully Connected");
+};
+
+let animation = {
+    playScoreBlue:  new CountUp('playScoreBlue', 0, 0, 0, .2, {useEasing: true, useGrouping: true,   separator: " ", decimal: "." }),
+    playScoreRed:  new CountUp('playScoreRed', 0, 0, 0, .2, {useEasing: true, useGrouping: true,   separator: " ", decimal: "." }),
+}
+
+socket.onclose = event => {
+    console.log("Socket Closed Connection: ", event);
+    socket.send("Client Closed!");
+};
+
+socket.onerror = error => {
+    console.log("Socket Error: ", error);
+};
+
+let teamSizeTemp;
+
+let bestOfTemp;
+let scoreVisibleTemp;
+let starsVisibleTemp;
+
+let tempImg;
+let tempMapName;
+let tempMapDiff;
+
+let scoreBlueTemp;
+let scoreRedTemp;
+let teamNameBlueTemp;
+let teamNameRedTemp;
+let gameState;
+
+socket.onmessage = event => {
+    let data = JSON.parse(event.data);
+	if(teamSizeTemp !== data.tourney.ipcClients.length) {
+		teamSizeTemp = data.tourney.ipcClients.length;
+	}
```

Comment:
unused variable Xd actually not sure why it exist

Reply:
It was previously used for counting the score manually, it seems that I forgot to delete that one, fix in a second.